### ISSUE TITLE
Blank UI screen with "Evaluate Variables" stage

### ISSUE THREAD

**Issue by Nirmalyasen** [spinnaker/spinnaker#5512]
### Issue Summary:
After execution of a pipeline with "Evaluate Variables" stage, clicking on the stage shows a blank page. 

### Cloud Provider(s):
NA

### Environment:
Spinnaker 1.17.5 running in K8S.

### Feature Area:
UI

### Steps to Reproduce:
Create a pipeline. Example pipeline json:

```
{
  "keepWaitingPipelines": false,
  "lastModifiedBy": "demoman",
  "limitConcurrent": true,
  "stages": [
    {
      "failOnFailedExpressions": true,
      "name": "Evaluate Variables",
      "refId": "1",
      "requisiteStageRefIds": [],
      "type": "evaluateVariables",
      "variables": [
        {
          "key": "key1",
          "value": "${trigger}"
        }
      ]
    }
  ],
  "triggers": [],
  "updateTs": "1582828612000"
}
```

Run the pipeline. Click on the run the stage. You get a blank page. 

Only way to get back is reload the page. Clicking around the visible menu makes no difference. 
<img width="1517" alt="Screen Shot 2020-02-27 at 10 51 24 AM" src="https://user-images.githubusercontent.com/25439242/75476457-7812ae00-594f-11ea-920e-12e1864874e0.png">

### Additional Details:
This error is seen in the browser console. 

```
index.js:3999 Automatically scrolling cursor into view after selection change this will be disabled in the next version set editor.$blockScrolling = Infinity to disable this message
warn @ index.js:3999
onCursorChange @ index.js:13314
EventEmitter._emit.EventEmitter._dispatchEvent @ index.js:3849
(anonymous) @ index.js:5421
EventEmitter._signal @ index.js:3865
setPosition @ index.js:7623
moveCursorTo @ index.js:5944
moveCursorFileStart @ index.js:5717
navigateFileStart @ index.js:14351
componentDidMount @ JsonEditor.tsx:49
Vh @ react-dom.production.min.js:216
Zh @ react-dom.production.min.js:220
(anonymous) @ react-dom.production.min.js:250
exports.unstable_runWithPriority @ scheduler.production.min.js:18
Fi @ react-dom.production.min.js:250
Di @ react-dom.production.min.js:249
Yh @ react-dom.production.min.js:248
Ii @ react-dom.production.min.js:251
Cd @ react-dom.production.min.js:85
angular.js:14961 executions still loading, skipping refresh
react-ga.js:88 [react-ga] ReactGA.initialize must be called first or GoogleAnalytics should be loaded manually
exports.default @ react-ga.js:88
internalGa @ react-ga.js:193
_gaCommand @ react-ga.js:213
send @ react-ga.js:328
event @ react-ga.js:531
ExecutionMarker.handleStageClick @ ExecutionMarker.tsx:57
(anonymous) @ react-dom.production.min.js:14
ja @ react-dom.production.min.js:15
(anonymous) @ react-dom.production.min.js:15
wa @ react-dom.production.min.js:17
Aa @ react-dom.production.min.js:18
ya @ react-dom.production.min.js:18
Da @ react-dom.production.min.js:21
Ad @ react-dom.production.min.js:84
Gi @ react-dom.production.min.js:250
Kb @ react-dom.production.min.js:39
Dd @ react-dom.production.min.js:86
(anonymous) @ react-dom.production.min.js:251
exports.unstable_runWithPriority @ scheduler.production.min.js:18
Ii @ react-dom.production.min.js:251
Cd @ react-dom.production.min.js:85
react-dom.production.min.js:198 Invariant Violation: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7BexecutionId%2C%20eventId%2C%20dryRun%2C%20rebake%2C%20type%2C%20user%2C%20parameters%2C%20enabled%2C%20artifacts%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1489103
    at x (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1489206)
    at Bf (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1535998)
    at https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1540755
    at S (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1547587)
    at Tg (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1555947)
    at bi (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1579241)
    at ci (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1579615)
    at Di (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1586453)
    at Yh (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1585840)
qh @ react-dom.production.min.js:198
Ch.c.callback @ react-dom.production.min.js:210
ih @ react-dom.production.min.js:193
hh @ react-dom.production.min.js:193
Vh @ react-dom.production.min.js:217
Zh @ react-dom.production.min.js:220
(anonymous) @ react-dom.production.min.js:250
exports.unstable_runWithPriority @ scheduler.production.min.js:18
Fi @ react-dom.production.min.js:250
Di @ react-dom.production.min.js:249
Yh @ react-dom.production.min.js:248
Xh @ react-dom.production.min.js:245
qf @ react-dom.production.min.js:243
enqueueSetState @ react-dom.production.min.js:130
E.setState @ react.production.min.js:13
updateStage @ StageExecutionDetails.tsx:199
(anonymous) @ StageExecutionDetails.tsx:203
TransitionHook.invokeHook @ transitionHook.js:92
(anonymous) @ transitionHook.js:77
TransitionHook.runAllHooks @ transitionHook.js:77
(anonymous) @ transition.js:582
(anonymous) @ angular.js:17330
(anonymous) @ angular.js:17378
$digest @ angular.js:18514
(anonymous) @ angular.js:18835
completeOutstandingRequest @ angular.js:6439
(anonymous) @ angular.js:6718
setTimeout (async)
Browser.self.defer @ angular.js:6716
$evalAsync @ angular.js:18833
(anonymous) @ angular.js:17202
scheduleProcessQueue @ angular.js:17378
$$resolve @ angular.js:17405
resolvePromise @ angular.js:17389
Deferred.resolve @ angular.js:17280
resolvePromise @ angular.js:12787
resolveHttpPromise @ angular.js:12768
(anonymous) @ angular.js:12774
completeRequest @ angular.js:13032
xhr.onload @ angular.js:12937
load (async)
(anonymous) @ angular.js:12920
(anonymous) @ angular.js:12720
(anonymous) @ angular.js:12461
(anonymous) @ angular.js:17330
(anonymous) @ angular.js:17378
$digest @ angular.js:18514
$apply @ angular.js:18903
(anonymous) @ angular.js:12775
completeRequest @ angular.js:13032
xhr.onload @ angular.js:12937
load (async)
(anonymous) @ angular.js:12920
(anonymous) @ angular.js:12720
(anonymous) @ angular.js:12461
(anonymous) @ angular.js:17330
(anonymous) @ angular.js:17378
$digest @ angular.js:18514
(anonymous) @ angular.js:18835
completeOutstandingRequest @ angular.js:6439
(anonymous) @ angular.js:6718
setTimeout (async)
Browser.self.defer @ angular.js:6716
$evalAsync @ angular.js:18833
(anonymous) @ angular.js:17202
scheduleProcessQueue @ angular.js:17378
then @ angular.js:17299
chainInterceptors @ angular.js:12389
$http @ angular.js:12376
(anonymous) @ ApiService.ts:106
getExecution @ execution.service.ts:103
hydrate @ execution.service.ts:478
Execution_Execution.toggleDetails @ Execution.tsx:138
ExecutionMarker.handleStageClick @ ExecutionMarker.tsx:58
(anonymous) @ react-dom.production.min.js:14
ja @ react-dom.production.min.js:15
(anonymous) @ react-dom.production.min.js:15
wa @ react-dom.production.min.js:17
Aa @ react-dom.production.min.js:18
ya @ react-dom.production.min.js:18
Da @ react-dom.production.min.js:21
Ad @ react-dom.production.min.js:84
Gi @ react-dom.production.min.js:250
Kb @ react-dom.production.min.js:39
Dd @ react-dom.production.min.js:86
(anonymous) @ react-dom.production.min.js:251
exports.unstable_runWithPriority @ scheduler.production.min.js:18
Ii @ react-dom.production.min.js:251
Cd @ react-dom.production.min.js:85
stateService.js:35 Transition Rejection($id: 2 type: 6, message: The transition errored, detail: Invariant Violation: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7BexecutionId%2C%20eventId%2C%20dryRun%2C%20rebake%2C%20type%2C%20user%2C%20parameters%2C%20enabled%2C%20artifacts%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. )
StateService._defaultErrorHandler @ stateService.js:35
TransitionHook.logError @ transitionHook.js:80
(anonymous) @ transitionHook.js:191
handleError @ transitionHook.js:94
TransitionHook.invokeHook @ transitionHook.js:107
(anonymous) @ transitionHook.js:77
TransitionHook.runAllHooks @ transitionHook.js:77
(anonymous) @ transition.js:582
(anonymous) @ angular.js:17330
(anonymous) @ angular.js:17378
$digest @ angular.js:18514
(anonymous) @ angular.js:18835
completeOutstandingRequest @ angular.js:6439
(anonymous) @ angular.js:6718
setTimeout (async)
Browser.self.defer @ angular.js:6716
$evalAsync @ angular.js:18833
(anonymous) @ angular.js:17202
scheduleProcessQueue @ angular.js:17378
$$resolve @ angular.js:17405
resolvePromise @ angular.js:17389
Deferred.resolve @ angular.js:17280
resolvePromise @ angular.js:12787
resolveHttpPromise @ angular.js:12768
(anonymous) @ angular.js:12774
completeRequest @ angular.js:13032
xhr.onload @ angular.js:12937
load (async)
(anonymous) @ angular.js:12920
(anonymous) @ angular.js:12720
(anonymous) @ angular.js:12461
(anonymous) @ angular.js:17330
(anonymous) @ angular.js:17378
$digest @ angular.js:18514
$apply @ angular.js:18903
(anonymous) @ angular.js:12775
completeRequest @ angular.js:13032
xhr.onload @ angular.js:12937
load (async)
(anonymous) @ angular.js:12920
(anonymous) @ angular.js:12720
(anonymous) @ angular.js:12461
(anonymous) @ angular.js:17330
(anonymous) @ angular.js:17378
$digest @ angular.js:18514
(anonymous) @ angular.js:18835
completeOutstandingRequest @ angular.js:6439
(anonymous) @ angular.js:6718
setTimeout (async)
Browser.self.defer @ angular.js:6716
$evalAsync @ angular.js:18833
(anonymous) @ angular.js:17202
scheduleProcessQueue @ angular.js:17378
then @ angular.js:17299
chainInterceptors @ angular.js:12389
$http @ angular.js:12376
(anonymous) @ ApiService.ts:106
getExecution @ execution.service.ts:103
hydrate @ execution.service.ts:478
Execution_Execution.toggleDetails @ Execution.tsx:138
ExecutionMarker.handleStageClick @ ExecutionMarker.tsx:58
(anonymous) @ react-dom.production.min.js:14
ja @ react-dom.production.min.js:15
(anonymous) @ react-dom.production.min.js:15
wa @ react-dom.production.min.js:17
Aa @ react-dom.production.min.js:18
ya @ react-dom.production.min.js:18
Da @ react-dom.production.min.js:21
Ad @ react-dom.production.min.js:84
Gi @ react-dom.production.min.js:250
Kb @ react-dom.production.min.js:39
Dd @ react-dom.production.min.js:86
(anonymous) @ react-dom.production.min.js:251
exports.unstable_runWithPriority @ scheduler.production.min.js:18
Ii @ react-dom.production.min.js:251
Cd @ react-dom.production.min.js:85
stateService.js:37 Invariant Violation: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7BexecutionId%2C%20eventId%2C%20dryRun%2C%20rebake%2C%20type%2C%20user%2C%20parameters%2C%20enabled%2C%20artifacts%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1489103
    at x (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1489206)
    at Bf (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1535998)
    at https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1540755
    at S (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1547587)
    at Tg (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1555947)
    at bi (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1579241)
    at ci (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1579615)
    at Di (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1586453)
    at Yh (https://demo.opsmx.com/vendor_R_Z~app.js?b4b90a5ac4082514a3e2:1:1585840)
<img width="1517" alt="Screen Shot 2020-02-27 at 10 51 24 AM" src="https://user-images.githubusercontent.com/25439242/75476285-3124b880-594f-11ea-84f3-716318e30b8d.png">

```

---

**Comment by maggieneterval** [comment]
@Nirmalyasen thanks for opening this issue. I think this was resolved in [1.17.6](https://www.spinnaker.io/community/releases/versions/#version-1176); can you please try upgrading and re-opening if you're still able to reproduce? Thanks!

**Comment by Nirmalyasen** [comment]
Yes, issue not seen in 1.18.1. Thanks for closing it.
